chart/catalog/templates/istio_catalog.yaml

```diff
@@ -53,4 +53,16 @@ spec:
         port:
           number: {{ .Values.service.servicePort }}
         subset: v1
+    retries:
+      attempts: 4
+      retryOn: 5xx
+
+---
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-{{ .Values.service.name }}-config
```

Comment:
This name actually conflicts with another config map we have in catalog.

Let's change it to `-mpftconfig`